examples/with-reason-react/src/server.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import express from 'express';
 import { renderToString } from 'react-dom/server';
 
-const App = require('../lib/js/src/app').comp; // BuckleScript output directory
+import App from '../lib/js/src/app'; // BuckleScript output directory
```

Comment:
We can change `"in-source"` in  bsconfig.json to `true` and just require it like normal, `import App from './App'`